### ISSUE TITLE
[Bug] Episodes not marked as viewed when using the library

### ISSUE THREAD

**Issue by notoco** [im85288/service.upnext#16]
Viewing a series from the library by the netflix plugin (using export to the library, not from the plugin), only the first episode is marked as viewed. The next ones are not marked. This problem does not occur when viewing directly from the netflix plugin.
Kodi 18 RC5 on windows and the same on last coreelec.

**Comment by im85288** [comment]
That’s not something in the realms of this add on as all it does is instruct the Netflix add on to play the episode. Marking of watched should be handled automatically via Kodi for any episode added to your library

**Comment by notoco** [comment]
Yes, I know ... I need to do some tests if it's not related to kodi 18, but I have not found this problem before ...

**Comment by im85288** [comment]
No worries, from what I read in the Kodi 18 rc5 announcement they know there will be bugs...so maybe this is one of them

**Comment by notoco** [comment]
https://forum.kodi.tv/showthread.php?tid=229951 - topic from 2015... great...

**Comment by im85288** [comment]
Ahh yes of course it’s the strm problems that have existed for a very long time! I think Emby has solved this. Not sure if you use it, but they have a Kodi add on (I used to be a developer for it) that syncs data to the Kodi dB. Worth a try :)

**Comment by notoco** [comment]
I just came to a similar conclusion after reading hundreds of pages on github.

**Comment by im85288** [comment]
Yes that will work fine, once you export your strm files Emby scans them in to get artwork etc..from that point in its like using Kodi natively
